tests/test_helpers.h

```diff
@@ -118,4 +119,24 @@ str_to_ull(const char *str, unsigned long long *value)
 
 	return 0;
 }
+
+/*
+ * get_granular_rand_size - (internal) generate random size value
+ * with specified granularity
+ */
+static size_t
+get_granular_rand_size(size_t val_max, size_t granularity)
```

Comment:
please don't add random tests.

Reply:
I think it's fine if we'll print the seed value.